elastic/kibana
# [Index Management] Disable toggling data retention in serverless

Fixes: https://github.com/elastic/kibana/issues/181242

## Summary

Disabling data retention for data streams in serverless should not be allowed. This PR introduces a plugin setting that serverless projects can configure in order to show/hide the toggle.

##### How to test
* Make sure that the toggle is hidden in serverless and data retention can be configured as normal
* Make sure that the toggle is visible in non-serverless installations
